### DISCUSSION TITLE
Keine Verbindung zum ESP32

### DISCUSSION THREAD

**Discussion by AndreasCr** [beabel/radiowecker#57] [Fragen & Antworten / Q&A]
Moin,
Ich habe mir den Bausatz bestellt, die Libraries installiert, die ESP32- Erweiterung installiert. Das alles mit der 2er IDE von Arduino. Wenn ich nun aber den ESP mit dem PC verbinde, dann findet der Computer keine Schnittstelle, auch wenn ich mit einem Chrome-Browser direkt Flaschen möchte, dann connected der nicht.
Was mache ich falsch ?
Gruß, Andreas

**Comment by beabel** [comment]
Es kann sein das du noch den richtigen Treiber benötigst.

auf [folgender Seite](https://www.ramser-elektro.at/support/downloads/treiber-downloads/) entweder den CH34x, oder den CP210x herunterladen und installieren

Bei manchen Boards muss man auch beim laden die Taste drücken damit es in den Transfer Modus geht.
1. Taste drücken (gedrückt halten)
2. einstecken
3. Transfer starten 
4. wenn Transfer gestartet kann Taste losgelassen werden

**Reply by AndreasCr** [reply]
Moin,
Du warst anscheinend auf dem richtigen Weg, aber wirklich geholfen hat mir dann erst die Origionl-Anleitung von AZ-Delivery. Die zeigt einen anderen Link zum Treiber (der auch funktioniert) und dort habe ich dann auch gelesen, dass die ESP-32-Library ab 3.0 nicht mehr sauber funktioniert und deswegen das Kompilieren abgebrochen wurde.
Also mein Tip : die Original-Doku lesen und befolgen !
Ansonsten Danke für Euere Engagement, super Projekt !
Gruß vom Niederrhein, Andreas

**Reply by beabel** [reply]
Das es zu Fehlern kommt ab V3 steht auch [hier.](https://github.com/beabel/radiowecker/discussions/50)

Das du diesen Fehler hast steht oben in deiner Frage jedoch nicht geschrieben, nur das keine Verbindung zum Board besteht. :-)

**Comment by AndreasCr** [comment]
Moin "kunigunde", Ich hatte den Code schon ohne Upload kompilieren lassen mit Fehlermeldung und bei der Suche nach den richtigen Treibern dann auch das mit dem V2-Fehler gelesen. Hast Du auch einen "echten" Namen, oder soll ich bei "Kunigunde" bleiben? 😉Gruß, Andreas--Gesendet mit der GMX Mail AppAm 18.08.24, 14:48 schrieb kunigunde ***@***.***>:

  
  Das es zu Fehlern kommt ab V3 steht auch hier.
  Das du diesen Fehler hast steht oben in deiner Frage jedoch nicht geschrieben, nur das keine Verbindung zum Board besteht. :-)
